wire-ios/Wire-iOS/Sources/UserInterface/Conversation/ConversationTitleView.swift

```diff
@@ -99,7 +108,7 @@ final class ConversationTitleView: TitleView {
 }
 
 extension NSTextAttachment {
-    static func verifiedShield() -> NSTextAttachment {
+    static func proteusVerifiedShield() -> NSTextAttachment {
         let attachment = NSTextAttachment()
         let shield = WireStyleKit.imageOfShieldverified
```

Comment:
Can we get rid of `WireStyleKit.imageOfShieldverified` and use the equivalent of `Assets.Images.` ?

Reply:
yes, done